plugins/uefi-db/README.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Plugin: UEFI db
+---
+
+## Introduction
+
+The EFI Signature Database (also known as `db`) contains multiple certificates, hashes and
+signatures, all signed by the KEK.
+
+It is used to allow signed binaries to be run with SecureBoot enabled. This plugin allows updating
+the `db` with new certificates issued by Microsoft or the OEM vendor.
+
+## Firmware Format
+
+The daemon will decompress the cabinet archive and extract a firmware blob in `EFI_SIGNATURE_LIST`
+format.
+See <https://www.uefi.org/sites/default/files/resources/UEFI%20Spec%202_6.pdf> for details.
+
+This plugin supports the following protocol ID:
+
+* `org.uefi.dbx2`
```

Comment:
shouldn't it be it's own new protocol?

Reply:
And for that matter isn't the code missing a declaration of the protocol?

Reply:
> org.uefi.squirt-a-blob-into-nvram-variable

Ah it really should have been:

```
org.uefi.autenticated-variable
```

But I guess there's not a value in renaming now.